pkgs/development/libraries/cctag/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, cmake
+, boost
+, eigen
+, opencv
+, tbb
+
+, avx2Support ? stdenv.hostPlatform.avx2Support
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cctag";
+  version = "1.0.3";
+
+  outputs = [ "lib" "dev" "out" ];
+
+  src = fetchFromGitHub {
+    owner = "alicevision";
+    repo = "CCTag";
+    rev = "v${version}";
+    hash = "sha256-foB+e7BCuUucyhN8FsI6BIT3/fsNLTjY6QmjkMWZu6A=";
+  };
+
+  cmakeFlags = [
+    # Feel free to create a PR to add CUDA support
+    "-DCCTAG_WITH_CUDA=OFF"
+
+    "-DCCTAG_ENABLE_SIMD_AVX2=${if avx2Support then "ON" else "OFF"}"
+
+    "-DCCTAG_BUILD_TESTS=ON"
+    "-DCCTAG_BUILD_APPS=OFF"
+  ];
+
+  patches = [
+    ./cmake-install-include-dir.patch
```

Comment:
It would be nice if the patch would get upstreamed.

Reply:
I'm not sure how upstreamable the patch is since I'm not sure if upstream's use of $<INSTALL_PREFIX> in generally OK here (i.e. whether this is a Nix-specific issue). Therefore I'm not looking into upstreaming the patch for now